app/src/androidTest/java/ch/epfl/sweng/swenggolf/AnswersTest.java

```diff
@@ -66,13 +67,19 @@
      * @param answer the message to be posted
      */
     public static void addAnswer(String answer) {
-        onView(withId(R.id.react_button)).perform(scrollTo(), click());
+        showOfferCustomScrollTo();
+        onView(withId(R.id.react_button)).perform(click());
         onView(withId(R.id.your_answer_description))
-                .perform(scrollTo(), typeText(answer), closeSoftKeyboard());
-        onView(withId(R.id.post_button)).perform(scrollTo(), click());
+                .perform(typeText(answer), closeSoftKeyboard());
+        onView(withId(R.id.post_button)).perform(click());
 
     }
 
+    public static void showOfferCustomScrollTo() {
```

Comment:
I see you are also using this method in other classes. You could move it to the TestUtility class so it is easier for other to find it.